_sass/_search-modal.scss

```diff
@@ -2,12 +2,14 @@
 .ui.dimmer.modals.page.inverted {
   background: rgba($color-white,.97);
   height: 100%;
-  overflow-y: visible;
+  overflow: hidden;
   .ui.basic.modal {
     &.search-modal {
       height: 100%;
       min-height: 100vh;
       transform: none;
+      overflow-y: scroll;
```

Comment:
I think you should change this to `overflow-y: auto;`. Overflow-y scroll will always show the scroll bar, in some browsers, even if the container doesn't need to scroll to see all the content. Auto works a little better where it shows the scroll bar if need be.

Reply:
That does work a lot nicer, I didn't know that. Thank you!

Reply:
![tenor-9](https://user-images.githubusercontent.com/5313708/101639997-f7183500-39fd-11eb-9478-85e3a63c9a18.gif)